util/timeutil/time_test.go

```diff
@@ -23,67 +23,65 @@ import (
 	"strings"
 	"testing"
 
-	. "github.com/pingcap/check"
+	"github.com/stretchr/testify/assert"
 )
 
-var _ = Suite(&testTimeSuite{})
+func TestGetTZNameFromFileName(t *testing.T) {
+	t.Parallel()
 
-func TestT(t *testing.T) {
-	TestingT(t)
-}
-
-type testTimeSuite struct{}
-
-func (s *testTimeSuite) TestgetTZNameFromFileName(c *C) {
 	tz, err := inferTZNameFromFileName("/usr/share/zoneinfo/Asia/Shanghai")
 
-	c.Assert(err, IsNil)
-	c.Assert(tz, Equals, "Asia/Shanghai")
+	assert.Nil(t, err)
```

Comment:
```suggestion
	require.NoError(t, err)
```

You may use more precious assertions. Also `require` plays the same as `c.Assert` which `FailNow` if assertion failed.

Reply:
ditto others.

Reply:
got it and updated